Core/src/main/java/com/plotsquared/core/plot/PlotId.java

```diff
@@ -193,52 +197,50 @@ public boolean equals(final Object obj) {
     }
 
     /**
-     * Get a String representation of the plot ID where the
-     * components are separated by ";"
+     * Returns a string representation of this Plot ID in the format "x;y".
      *
-     * @return {@code x + ";" + y}
+     * @return a string representation of this Plot ID
```

Comment:
I'd show a "human-friendly" representation still, in case the wording may be unclear.

Reply:
I’ll add this in the description. It’s probably a better spot for it.